-------------------
CSE247 Robot tester starts testing heaps on 2019-10-13 19:56:22 -0500

========
Testing in ./GradingRepos/repos/labs-lucharles: heaps as of 2019-10-05 00:05
========
Now in: /home/coles/git/cse247/cse247-support-f19/GradingRepos/repos/labs-lucharles
Your branch is up-to-date with 'origin/master'.
HEAD is now at b70e4e2 robograde
No local changes to save

Already up-to-date.
Going back in time to: 2019-10-05 00:05
At hash 291d7fcd4840b030c87c8df5e5bc07f6d67a8c15

Buildfile: /home/coles/git/cse247/cse247-support-f19/build.xml

copytest:
     [echo] would copy from /home/coles/git/cse247/cse247-f19-graders/ key lucharles testfilesource test/TestInsert package heaps repo ./GradingRepos/repos/labs-lucharles
     [copy] Copying 1 file to /home/coles/git/cse247/cse247-support-f19/GradingRepos/repos/labs-lucharles/labs/heaps/test

BUILD SUCCESSFUL
Total time: 0 seconds
Compiling the code
Buildfile: /home/coles/git/cse247/cse247-support-f19/build.xml

compile:
     [echo] Compiling heaps/MinHeap
    [javac] Compiling 1 source file to /home/coles/git/cse247/cse247-support-f19/GradingRepos/repos/labs-lucharles/bin

BUILD SUCCESSFUL
Total time: 1 second
Buildfile: /home/coles/git/cse247/cse247-support-f19/build.xml

compile:
     [echo] Compiling heaps/test/TestInsert
    [javac] Compiling 1 source file to /home/coles/git/cse247/cse247-support-f19/GradingRepos/repos/labs-lucharles/bin

BUILD SUCCESSFUL
Total time: 1 second
Running test test.TestInsert
Buildfile: /home/coles/git/cse247/cse247-support-f19/build.xml

test:
    [junit] Running heaps.test.TestInsert
    [junit] Testsuite: heaps.test.TestInsert
    [junit] Testing with heap size: 50
    [junit] ===== Failure report for testInsert(heaps.test.TestInsert) ========
    [junit] Failure message: Heap Validation Error (detail shown below):
    [junit] java.lang.Error: I found a null entry within the currently occupied portion of the heap, at index 1
    [junit] Tree before the problem occurred:
    [junit] 
    [junit] What went wrong: I found a null entry within the currently occupied portion of the heap, at index 1
    [junit] Tree that triggered this problem:
    [junit]  null
    [junit] 
    [junit] java.lang.Error: I found a null entry within the currently occupied portion of the heap, at index 1
    [junit] ===== End Failure report for testInsert(heaps.test.TestInsert) ========
    [junit] 	at heaps.validate.MinHeapValidator.checkForGaps(MinHeapValidator.java:113)
    [junit] 
    [junit] 	at heaps.validate.MinHeapValidator.check(MinHeapValidator.java:30)
    [junit] 
    [junit] 	at heaps.test.TestInsert.testInsert(TestInsert.java:40)
    [junit] ===== Failure report for testLocUpdatedDecrease(heaps.test.TestInsert) ========
    [junit] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    [junit] Failure message: Heap Validation Error (detail shown below):
    [junit] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    [junit] java.lang.Error: I found a null entry within the currently occupied portion of the heap, at index 1
    [junit] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    [junit] Tree before the problem occurred:
    [junit] 	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
    [junit] 
    [junit] 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
    [junit] What went wrong: I found a null entry within the currently occupied portion of the heap, at index 1
    [junit] 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
    [junit] Tree that triggered this problem:
    [junit] 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
    [junit]  null
    [junit] 
    [junit] ===== End Failure report for testLocUpdatedDecrease(heaps.test.TestInsert) ========
    [junit] 	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
    [junit] 	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
    [junit] 	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
    [junit] 	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
    [junit] 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
    [junit] 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
    [junit] 	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
    [junit] 	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
    [junit] 	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
    [junit] 	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
    [junit] 	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
    [junit] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
    [junit] 	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:38)
    [junit] 	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:535)
    [junit] 	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1182)
    [junit] 	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:1033)
    [junit] java.lang.Error: I found a null entry within the currently occupied portion of the heap, at index 1
    [junit] 	at heaps.validate.MinHeapValidator.checkForGaps(MinHeapValidator.java:113)
    [junit] 	at heaps.validate.MinHeapValidator.check(MinHeapValidator.java:30)
    [junit] 	at heaps.test.TestInsert.testLocUpdatedDecrease(TestInsert.java:55)
    [junit] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    [junit] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    [junit] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    [junit] 	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
    [junit] 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
    [junit] 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
    [junit] 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
    [junit] 	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
    [junit] 	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
    [junit] 	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
    [junit] 	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
    [junit] 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
    [junit] 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
    [junit] 	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
    [junit] 	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
    [junit] 
    [junit] 
    [junit] Tests run: 3, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 0.139 sec
    [junit] 	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
    [junit] Output:
    [junit] 	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
    [junit] Testing with heap size: 50
    [junit] 	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
    [junit] ===== Failure report for testInsert(heaps.test.TestInsert) ========
    [junit] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
    [junit] Failure message: Heap Validation Error (detail shown below):
    [junit] 	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:38)
    [junit] java.lang.Error: I found a null entry within the currently occupied portion of the heap, at index 1
    [junit] 	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:535)
    [junit] Tree before the problem occurred:
    [junit] 
    [junit] What went wrong: I found a null entry within the currently occupied portion of the heap, at index 1
    [junit] Tree that triggered this problem:
    [junit]  null
    [junit] 
    [junit] ===== End Failure report for testInsert(heaps.test.TestInsert) ========
    [junit] 
    [junit] 
    [junit] ===== Failure report for testLocUpdatedDecrease(heaps.test.TestInsert) ========
    [junit] 	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1182)
    [junit] 	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:1033)
    [junit] Failure message: Heap Validation Error (detail shown below):
    [junit] java.lang.Error: I found a null entry within the currently occupied portion of the heap, at index 1
    [junit] Tree before the problem occurred:
    [junit] 
    [junit] What went wrong: I found a null entry within the currently occupied portion of the heap, at index 1
    [junit] Tree that triggered this problem:
    [junit]  null
    [junit] 
    [junit] ===== End Failure report for testLocUpdatedDecrease(heaps.test.TestInsert) ========
    [junit] 
    [junit] 
    [junit] 
    [junit] Error: 
    [junit] java.lang.Error: I found a null entry within the currently occupied portion of the heap, at index 1
    [junit] 	at heaps.validate.MinHeapValidator.checkForGaps(MinHeapValidator.java:113)
    [junit] 	at heaps.validate.MinHeapValidator.check(MinHeapValidator.java:30)
    [junit] 	at heaps.test.TestInsert.testInsert(TestInsert.java:40)
    [junit] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    [junit] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    [junit] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    [junit] 	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
    [junit] 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
    [junit] 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
    [junit] 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
    [junit] 	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
    [junit] 	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
    [junit] 	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
    [junit] 	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
    [junit] 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
    [junit] 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
    [junit] 	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
    [junit] 	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
    [junit] 	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
    [junit] 	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
    [junit] 	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
    [junit] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
    [junit] 	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:38)
    [junit] 	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:535)
    [junit] 	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1182)
    [junit] 	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:1033)
    [junit] java.lang.Error: I found a null entry within the currently occupied portion of the heap, at index 1
    [junit] 	at heaps.validate.MinHeapValidator.checkForGaps(MinHeapValidator.java:113)
    [junit] 	at heaps.validate.MinHeapValidator.check(MinHeapValidator.java:30)
    [junit] 	at heaps.test.TestInsert.testLocUpdatedDecrease(TestInsert.java:55)
    [junit] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    [junit] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    [junit] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    [junit] 	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
    [junit] 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
    [junit] 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
    [junit] 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
    [junit] 	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
    [junit] 	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
    [junit] 	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
    [junit] 	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
    [junit] 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
    [junit] 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
    [junit] 	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
    [junit] 	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
    [junit] 	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
    [junit] 	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
    [junit] 	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
    [junit] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
    [junit] 	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:38)
    [junit] 	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:535)
    [junit] 	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1182)
    [junit] 	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:1033)
    [junit] 
    [junit] Tests run: 3, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 0.139 sec
    [junit] ------------- Standard Output ---------------
    [junit] Testing with heap size: 50
    [junit] ===== Failure report for testInsert(heaps.test.TestInsert) ========
    [junit] Failure message: Heap Validation Error (detail shown below):
    [junit] java.lang.Error: I found a null entry within the currently occupied portion of the heap, at index 1
    [junit] Tree before the problem occurred:
    [junit] 
    [junit] What went wrong: I found a null entry within the currently occupied portion of the heap, at index 1
    [junit] Tree that triggered this problem:
    [junit]  null
    [junit] 
    [junit] ===== End Failure report for testInsert(heaps.test.TestInsert) ========
    [junit] 
    [junit] 
    [junit] ===== Failure report for testLocUpdatedDecrease(heaps.test.TestInsert) ========
    [junit] Failure message: Heap Validation Error (detail shown below):
    [junit] java.lang.Error: I found a null entry within the currently occupied portion of the heap, at index 1
    [junit] Tree before the problem occurred:
    [junit] 
    [junit] What went wrong: I found a null entry within the currently occupied portion of the heap, at index 1
    [junit] Tree that triggered this problem:
    [junit]  null
    [junit] 
    [junit] ===== End Failure report for testLocUpdatedDecrease(heaps.test.TestInsert) ========
    [junit] 
    [junit] 
    [junit] ------------- ---------------- ---------------
    [junit] ------------- Standard Error -----------------
    [junit] java.lang.Error: I found a null entry within the currently occupied portion of the heap, at index 1
    [junit] 	at heaps.validate.MinHeapValidator.checkForGaps(MinHeapValidator.java:113)
    [junit] 	at heaps.validate.MinHeapValidator.check(MinHeapValidator.java:30)
    [junit] 	at heaps.test.TestInsert.testInsert(TestInsert.java:40)
    [junit] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    [junit] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    [junit] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    [junit] 	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
    [junit] 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
    [junit] 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
    [junit] 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
    [junit] 	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
    [junit] 	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
    [junit] 	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
    [junit] 	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
    [junit] 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
    [junit] 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
    [junit] 	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
    [junit] 	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
    [junit] 	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
    [junit] 	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
    [junit] 	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
    [junit] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
    [junit] 	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:38)
    [junit] 	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:535)
    [junit] 	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1182)
    [junit] 	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:1033)
    [junit] java.lang.Error: I found a null entry within the currently occupied portion of the heap, at index 1
    [junit] 	at heaps.validate.MinHeapValidator.checkForGaps(MinHeapValidator.java:113)
    [junit] 	at heaps.validate.MinHeapValidator.check(MinHeapValidator.java:30)
    [junit] 	at heaps.test.TestInsert.testLocUpdatedDecrease(TestInsert.java:55)
    [junit] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    [junit] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    [junit] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    [junit] 	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
    [junit] 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
    [junit] 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
    [junit] 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
    [junit] 	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
    [junit] 	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
    [junit] 	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
    [junit] 	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
    [junit] 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
    [junit] 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
    [junit] 	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
    [junit] 	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
    [junit] 	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
    [junit] 	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
    [junit] 	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
    [junit] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
    [junit] 	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:38)
    [junit] 	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:535)
    [junit] 	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1182)
    [junit] 	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:1033)
    [junit] ------------- ---------------- ---------------
    [junit] 
    [junit] Testcase: testEmptyHeap took 0.029 sec
    [junit] Testcase: testInsert took 0.074 sec
    [junit] 	Caused an ERROR
    [junit] null
    [junit] Heap Validation Error (detail shown below):
    [junit] java.lang.Error: I found a null entry within the currently occupied portion of the heap, at index 1
    [junit] Tree before the problem occurred:
    [junit] 
    [junit] What went wrong: I found a null entry within the currently occupied portion of the heap, at index 1
    [junit] Tree that triggered this problem:
    [junit]  null
    [junit] 
    [junit] 	at heaps.validate.MinHeapValidator.check(MinHeapValidator.java:43)
    [junit] 	at heaps.test.TestInsert.testInsert(TestInsert.java:40)
    [junit] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    [junit] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    [junit] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    [junit] 
    [junit] Testcase: testLocUpdatedDecrease took 0.004 sec
    [junit] 	Caused an ERROR
    [junit] null
    [junit] Heap Validation Error (detail shown below):
    [junit] java.lang.Error: I found a null entry within the currently occupied portion of the heap, at index 1
    [junit] Tree before the problem occurred:
    [junit] 
    [junit] What went wrong: I found a null entry within the currently occupied portion of the heap, at index 1
    [junit] Tree that triggered this problem:
    [junit]  null
    [junit] 
    [junit] 	at heaps.validate.MinHeapValidator.check(MinHeapValidator.java:43)
    [junit] 	at heaps.test.TestInsert.testLocUpdatedDecrease(TestInsert.java:55)
    [junit] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    [junit] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    [junit] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    [junit] 

BUILD FAILED
/home/coles/git/cse247/cse247-support-f19/build.xml:34: Test heaps.test.TestInsert failed

Total time: 1 second
========
Testing in ./GradingRepos/repos/labs-lucharles: heaps as of 2019-10-12 00:05
========
Now in: /home/coles/git/cse247/cse247-support-f19/GradingRepos/repos/labs-lucharles
Your branch is up-to-date with 'origin/master'.
HEAD is now at b70e4e2 robograde
No local changes to save

Already up-to-date.
Going back in time to: 2019-10-12 00:05
At hash c7fcc554fd59719e8e3fc01881ddf1c628f7f5d1

Buildfile: /home/coles/git/cse247/cse247-support-f19/build.xml

copytest:
     [echo] would copy from /home/coles/git/cse247/cse247-f19-graders/ key lucharles testfilesource test/TestInsert package heaps repo ./GradingRepos/repos/labs-lucharles
     [copy] Copying 1 file to /home/coles/git/cse247/cse247-support-f19/GradingRepos/repos/labs-lucharles/labs/heaps/test

BUILD SUCCESSFUL
Total time: 0 seconds
Compiling the code
Buildfile: /home/coles/git/cse247/cse247-support-f19/build.xml

compile:
     [echo] Compiling heaps/MinHeap
    [javac] Compiling 1 source file to /home/coles/git/cse247/cse247-support-f19/GradingRepos/repos/labs-lucharles/bin

BUILD SUCCESSFUL
Total time: 1 second
Buildfile: /home/coles/git/cse247/cse247-support-f19/build.xml

compile:
     [echo] Compiling heaps/test/TestInsert
    [javac] Compiling 1 source file to /home/coles/git/cse247/cse247-support-f19/GradingRepos/repos/labs-lucharles/bin

BUILD SUCCESSFUL
Total time: 2 seconds
Running test test.TestInsert
Buildfile: /home/coles/git/cse247/cse247-support-f19/build.xml

test:
    [junit] Running heaps.test.TestInsert
    [junit] Testsuite: heaps.test.TestInsert
    [junit] Testing with heap size: 43
    [junit] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.189 sec
    [junit] Output:
    [junit] Testing with heap size: 43
    [junit] 
    [junit] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.189 sec
    [junit] ------------- Standard Output ---------------
    [junit] Testing with heap size: 43
    [junit] ------------- ---------------- ---------------
    [junit] 
    [junit] Testcase: testEmptyHeap took 0.046 sec
    [junit] Testcase: testInsert took 0.092 sec
    [junit] Testcase: testLocUpdatedDecrease took 0.008 sec

BUILD SUCCESSFUL
Total time: 1 second
================
Lab is counted as working
================

CSE247 Robot tester ends testing heaps on 2019-10-13 19:56:36 -0500
-------------------
